src/coreclr/src/hosts/corerun/corerun.cpp

```diff
@@ -562,7 +558,6 @@ HRESULT InitializeHost(
         "APP_PATHS",
         "APP_NI_PATHS",
         "NATIVE_DLL_SEARCH_DIRECTORIES",
-        "APP_LOCAL_WINMETADATA",
```

Comment:
>APP_LOCAL_WINMETADATA [](start = 9, length = 21)

We might want to add a log or something that when this is found we provide a warning. Is that feasible?